applications/nrf_desktop/board_configuration.rst

```diff
@@ -111,7 +111,7 @@ Sample mouse or keyboard (``nrf54l15dk/nrf54l05/cpuapp``)
       * The build types allow to build the application as a mouse or a keyboard.
       * Inputs are simulated based on the hardware button presses.
       * On the nRF54L05 SoC, you can only use the **GPIO1** port for PWM hardware peripheral output.
-        Because of that, on the DK PCA10156 revision v0.8.1, **LED 0** and **LED 2** cannot be used for PWM output.
+        Because of that, on the DK PCA10156 revision v0.9.3, **LED 0** and **LED 2** cannot be used for PWM output.
```

Comment:
We officially support DK in the following version ranger v0.9.1 - v0.9.3. Are you sure that you just want to mention the newest one here?

Reply:
I was wondering a bit about changing it to `v0.9.x` (to make it more generic). Still, if a new DK revision is released, we might end up with invalid/outdated information here. That's why I used `v0.9.3` here as an example (referring to the newest DK revision in docs).

Do you have a suggestion on how should we change it?